keg-client/src/Taps/BeerBlock.tsx

```diff
@@ -86,6 +97,35 @@ function EmptyOverlay() {
   );
 }
 
+export interface DetailDialogProps {
+  beer: Beer;
+  isOpen: boolean;
+  onClose: () => void;
+}
+
+function DetailDialog(props: DetailDialogProps) {
+  const { beer, isOpen, onClose } = props;
+  return (
+    <Dialog onClose={onClose} aria-labelledby="dialog-title" open={isOpen}>
+      <DialogTitle id="dialog-title">
+        <Typography gutterBottom variant="h5" component="h2" align="center">
+          {beer.name}
+        </Typography>
+      </DialogTitle>
+      <DialogContent>
+        <Typography variant="body1" component="p" gutterBottom>
+          {beer.description ? beer.description : "No Description Provided."}
```

Comment:
`{ beer.description || "No Description Provided" }` is a common javascript idiom you could use in the future - `\\` returns the actual arguments, not `true` or `false`